packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -9,7 +9,8 @@ environment:
 
 dependencies: {{#enable_dev_menu}}
   alice: ^1.0.0-dev.10
-  alice_dio: ^1.0.4{{/enable_dev_menu}}
+  alice_dio: ^1.0.4{{/enable_dev_menu}}{{#enable_patrol}}
+  args: ^2.5.0{{/enable_patrol}}
```

Comment:
Why we need to add the `args` package to the `dependencies` instead of into the `dependencies` ?

Reply:
When adding it in dev_dependencies I get the following error,
`The imported package 'args' isn't a dependency of the importing package.
Try adding a dependency for 'args' in the 'pubspec.yaml' file.dart[depend_on_referenced_packages](https://dart.dev/diagnostics/depend_on_referenced_packages)`

Reply:
ok let me as in another way. Where this package is used and do we really need it ?

Reply:
Technically we could rewrite it to not use Args package

Reply:
In this case please remote the args dependency.

Reply:
Args dependency has been removed